mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -65,13 +68,17 @@ public static Table loadTable(Configuration conf, Properties props) {
   private static Table loadTable(Configuration conf, String tableIdentifier, String tableLocation) {
     Optional<Catalog> catalog = loadCatalog(conf);
 
+    Table table;
     if (catalog.isPresent()) {
       Preconditions.checkArgument(tableIdentifier != null, "Table identifier not set");
-      return catalog.get().loadTable(TableIdentifier.parse(tableIdentifier));
+      table = catalog.get().loadTable(TableIdentifier.parse(tableIdentifier));
+    } else {
+      Preconditions.checkArgument(tableLocation != null, "Table location not set");
+      table = new HadoopTables(conf).load(tableLocation);
     }
 
-    Preconditions.checkArgument(tableLocation != null, "Table location not set");
-    return new HadoopTables(conf).load(tableLocation);
+    LOG.info("Table loaded by catalog: [{}]", table);
```

Comment:
I think this should be done in the catalog or `HadoopTables`, not in `Catalogs`. Otherwise, we don't get the log message consistently when table loads go directly through the catalog (as in Spark 3). And if we were to later add a message to the catalog, then this would be a duplicate.

Reply:
Done